Update 02-average.Rmd (#5)
deynarde/kag-knight-guide: 02-average.Rmd

  <img src="imgs/knightmodel.png">

The hitbox of a knight is an octagon.

Both jab and slash are shaped as arcs, which move with your character and are active for a certain duration (slashes last much longer than jabs).

> Your shield covers an arc around your character centred on the direction you point your mouse. If you were hit through your shield by someone – ask yourself if you were pointing in the right direction to cover the full arc of the attack.

Reviewer: ```suggestion
Your shield covers an arc around your character centered on the direction pointed by your mouse. If you were hit through your shield by someone – ask yourself if you were aiming at the right direction to cover the full arc of the attack.
```

<video style="display:block; max-width:100%; height:auto;" src="webm/02-good/20.webm" controls loop="loop" preload></video><br>

Keep in mind that your jumps should be small. A very common mistake people make is that they jump too far - jumping too high while slashing makes your slash range worse than if you would actually not jump at all (slashing below you when you're going up has a very small range), making it easier for enemies to slash you from below.

---
These are the 3 most important things at this level: slashing fast, jabbing after successful slashes and performing low jumps while slashing. Train these until it feels natural and you do it instinctively.
---
Additionally, there are a few more things you can keep in mind, such as:

### Using double slashes

Another common mistake is just constantly single slashing a shielding enemy. Some people use double slashes too rarely. Double slashes are a very good counter against defensive playstyles. Additionally, you can always land a jab after connecting with a double slash:

<video style="display:block; max-width:100%; height:auto;" src="webm/02-good/22.webm" controls loop="loop" preload></video><br>

> Be wary, double slashing can be countered by good players, but you'll learn more about it this in the next chapter.

Reviewer: ```suggestion
Be wary, double slashing can be countered by good players, but you'll learn more about this in the next chapter.
```

Jumping over the shield sliding enemy whilst charging a slash and then hitting them in the back (either with the charged slash or jab) is a consistent way to beat shield bashers. In some of the situations, the enemy won't even be able to put his shield back up, no matter what -- if you're shield sliding really fast there is a delay between sliding and shielding. However, this doesn't happen every time.

[gif 36 - overhead slash]

You can also do an overhead jab if you don't have enough time to charge a slash:

<video style="display:block; max-width:100%; height:auto;" src="webm/02-good/37.webm" controls loop="loop" preload></video><br>

Keep in mind that in this variation, if the enemy manages to put his shield back up, you may get stunned due to jabbing his shield.

Again, don't be demotivated if this may seem a lot and it is hard for you to learn everything. Many good knights struggled with this - it takes them a while before it clicks and shield bashes just feel natural to them.

## Playing CTF

> At this point it would be good fir you to read the section for [Playing CTF] as well.

Reviewer: ```suggestion
At this point it would be good for you to read the section for [Playing CTF] as well.
```
